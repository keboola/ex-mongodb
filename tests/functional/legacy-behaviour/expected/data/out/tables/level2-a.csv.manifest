{"incremental":false,"columns":["count","itemId","level1_pk"],"column_metadata":{"count":[{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"string"}],"itemId":[{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"string"}],"level1_pk":[{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"string"}]}}