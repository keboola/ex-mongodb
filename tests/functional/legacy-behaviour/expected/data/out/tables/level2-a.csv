"1","123","d65f09a7f75f31ad639f3c2b1d7d4d3d"
"1","123","d65f09a7f75f31ad639f3c2b1d7d4d3d"
"20","456","77297cc0f4e8f0ea5641dd4b688b331f"
"20","456","77297cc0f4e8f0ea5641dd4b688b331f"
